core/src/main/java/com/novoda/noplayer/drm/DownloadDrmSessionCreator.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.noplayer.drm;
```

Comment:
We were planning on going back through the project when the work of migrating is done to flatten the package structure and make it harder to get at some of the internal dependencies.